lib/process-services/task-list/components/task-list.component.ts

```diff
@@ -50,7 +50,9 @@ export class TaskListComponent implements OnChanges, AfterContentInit, Paginated
     @Input()
     processInstanceId: string;
 
-    /** The Definition Key of the process. */
+    /** The Definition Key of the process.
+     * @deprecated
```

Comment:
can you add it as separate comment with the version number like that:

    /** @deprecated 2.4.0 */
 thanks